### ISSUE TITLE
[question]: Remove menubar entries

### ISSUE THREAD

**Issue by Tectu** [jgazeau/shadocs#249]
### Question?

Is there a way to remove the `QR Code` and `About` entries in the top menu bar?

**Comment by jgazeau** [comment]
Hi @Tectu ,
For now there is no such feature. 
Tho, you could create a css to disable the display of these button, but it would make sense to have a theme option that manage display of all buttons in the navbar.
I'll add this feature in the future

**Comment by Tectu** [comment]
I think that adding theme options to control the visibility of these buttons is indeed a good feature to add :)

Looking forward to that - Thanks!

**Comment by jgazeau** [comment]
Hi @Tectu ,
I've add some options you can set within the `params` section of your site configuration file.
```
navbarPrintEntry        = false
navbarQRCodeEntry       = false
navbarSchortcutsEntry   = false
navbarInfo              = false
```
By default all buttons are displayed but you can now disable them if you wish.
Solved with #258

**Comment by Tectu** [comment]
Thank you very much!

I hope you won't abandon this theme any time soon. After searching & testing different hugo themes for a technical documentation page for several days I eventually settled with shadocs - Looks very promising!

It's a very straight forward, no-shenanigans theme that is well suited for large documentation projects without relying on bloat.
